adoc/reference-webui-admin.adoc

```diff
@@ -633,6 +633,12 @@ Calls the [command]``mgr-register`` command, which synchronizes client registrat
 menu:mgr-sync-refresh-default:[]::
 the default time at which the start of synchronization with SUSE Customer Center (SCC) takes place (``mgr-sync-refresh``).
 
+menu:minion-action-cleanup-default:[]::
+deletes stale minion action data from the file system.
+First it completes the unfinished action.
+An unfinished action can occur if the server has missed the results of the action.
+Then after action completion it removes artifacts such as executed script files.
```

Comment:
Usually the unfinished actions found here and those that we are deleting artifacts for would not be the same ones. Actions are attempted to be completed only in exceptional cases (we missed the result for some reason), while any successful script execution needs to be cleaned up afterwards (script removal). Maybe we could make it more clear like for example:

"First it tries to complete any possibly unfinished actions by looking up corresponding results (from the Salt job cache). An unfinished action can occur if the server has missed the results of the action. For successfully completed actions it removes artifacts such as executed script files."